build.gradle.kts

```diff
@@ -28,15 +28,15 @@ buildscript {
     }
 }
 
-val annotationProcessorVersion: String by project
+val edcGradlePluginsVersion: String by project
 
 allprojects {
     apply(plugin = "${group}.edc-build")
 
     // configure which version of the annotation processor to use. defaults to the same version as the plugin
     configure<org.eclipse.edc.plugins.autodoc.AutodocExtension> {
         outputDirectory.set(project.layout.buildDirectory.asFile)
-        processorVersion.set(annotationProcessorVersion)
+        processorVersion.set(edcGradlePluginsVersion)
```

Comment:
this could also be not set, by default it will be same as the project version (0.11.0-SNAPSHOT)